octoprint_FriendlyBeeper/static/js/FriendlyBeeper.js

```diff
@@ -34,7 +34,14 @@ $(function() {
         };
 
         self.beep_test = function() {
-            OctoPrint.simpleApiCommand(PLUGIN_ID, "beep_test")
+            OctoPrint.simpleApiCommand(PLUGIN_ID, "beep_test", {
+                beep_method: self.beep_method(),
```

Comment:
The indentation seems odd. Also, I prefer a `get_settings` or `toJSON` method for things like this to hold all the code for this, then potentially duplicate this.

Reply:
oh, sorry, will fix that.

About the code duplication, yea, that's something I noticed too. I'll try to fix that here and I also got an idea how to do that for the python code too.